bundler/lib/bundler/gem_helper.rb

```diff
@@ -2,9 +2,14 @@
 
 require_relative "../bundler"
 require "shellwords"
+require "digest/sha2"
```

Comment:
Can we keep lazily loading `digest/sha2`?

Reply:
lazy loading prevents using this map:
```
    CHECKSUMS = {
      "sha256" => ::Digest::SHA256,
      "sha512" => ::Digest::SHA512,
    }.freeze
```
But I can use strings and const_get instead.